@xen-orchestra/lite/src/components/pool/network/PoolNetworksSidePanel.vue

```diff
@@ -0,0 +1,235 @@
+                  :icon="faCopy"
+                  @click="copy(networkNbd)"
+                />
+              </template>
+            </VtsCardRowKeyValue>
+            <!-- DEFAULT LOCKING MODE -->
+            <VtsCardRowKeyValue>
+              <template #key>{{ $t('locking-mode-default') }}</template>
+              <template #value>{{ networkDefaultLockingMode }}</template>
+            </VtsCardRowKeyValue>
+          </div>
+        </UiCard>
+        <UiCard v-if="network?.PIFs?.length > 0" class="card-container">
+          <div class="typo p1-medium">
+            {{ $t('pifs') }}
+            <UiCounter v-if="pifsCount" :value="pifsCount" variant="primary" size="small" accent="neutral" />
+          </div>
+          <table class="simple-table">
+            <thead>
+              <tr>
+                <th class="text-left typo p3-regular">
+                  {{ $t('host') }}
+                </th>
+                <th class="text-left typo p3-regular">
+                  {{ $t('device') }}
+                </th>
+                <th class="text-left typo p3-regular">
+                  {{ $t('pifs-status') }}
+                </th>
+                <th />
+              </tr>
+            </thead>
+            <tbody>
+              <PifRow v-for="pif in pifs" :key="pif.uuid" :pif />
+            </tbody>
+          </table>
+        </UiCard>
+      </template>
+    </template>
+  </UiPanel>
+</template>
+
+<script setup lang="ts">
+import PifRow from '@/components/pif/PifRow.vue'
+import { useNetworkStore } from '@/stores/xen-api/network.store'
+import { usePifStore } from '@/stores/xen-api/pif.store'
+import VtsCardRowKeyValue from '@core/components/card/VtsCardRowKeyValue.vue'
+import VtsNoSelectionHero from '@core/components/state-hero/VtsNoSelectionHero.vue'
+import UiButton from '@core/components/ui/button/UiButton.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiCardTitle from '@core/components/ui/card-title/UiCardTitle.vue'
+import UiCounter from '@core/components/ui/counter/UiCounter.vue'
+import UiPanel from '@core/components/ui/panel/UiPanel.vue'
+import { useRouteQuery } from '@core/composables/route-query.composable'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { toArray } from '@core/utils/to-array.utils'
+import { faCopy, faEdit, faEllipsis, faTrash } from '@fortawesome/free-solid-svg-icons'
+import { useClipboard } from '@vueuse/core'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { records: networks } = useNetworkStore().subscribe()
+const { pifsByNetwork } = usePifStore().subscribe()
+const { t } = useI18n()
+
+const networkId = useRouteQuery('id')
+const network = computed(() => networks.value.find(network => network.uuid === networkId.value))
+
+const pifs = computed(() => (network.value ? toArray(pifsByNetwork.value.get(network.value.$ref)) : []))
```

Comment:
I don't think it's really relevant to use a Map to find the PIFs of a network.

I think a simple function in the store would be enough:

```ts
function getByNetworkRef(networkRef: XenApiNetwork['$ref']) {
  return baseContext.records.value.filter(pif => pif.network === networkRef)
}
```

Then something like this:

```suggestion
const pifs = computed(() => getByNetworkRef(network.$ref))
```